src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -437,4 +437,105 @@ object Utilities {
     })
     semanticList.toList
   }
+
+    def reportUnresolvedMethods(xtocpg: Try[Cpg], filename: String): Unit = {
```

Comment:
@gaurav-gogia Remove the function from `Utilities.scala`and create a new file `unresolvedReportUtility.scala` and have this code there